heartbeat/magefile.go

```diff
@@ -86,3 +86,10 @@ func FieldDocs() error {
 func Config() error {
 	return devtools.Config(devtools.AllConfigTypes, heartbeat.ConfigFileParams(), ".")
 }
+
+// AssembleDarwinUniversal merges the darwin/amd64 and darwin/arm64 into a single
+// universal binary using `lipo`. It assumes the darwin/amd64 and darwin/arm64
+// were built and only performs the merge.
+func AssembleDarwinUniversal() error {
```

Comment:
This looks like it could be added to `github.com/elastic/beats/v7/dev-tools/mage/target/build`. If so, then any of the Beats that are directly using `mage:import` of that package won't need to duplicate it.